src/lib/components/Switch/root.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts" context="module">
+	import type { BaseProps } from '$lib/types';
+	import { uniqueContext } from '$lib/helpers/uniqueContext';
+
+	export type SwitchProps = BaseProps<HTMLButtonElement> & {
+		checked?: boolean;
+		defaultChecked?: boolean;
```

Comment:
@TGlide and I discussed if we want to have these in the lib. Right now we had concluded that theres no real reason to have them in svelte land, since it more exists to make react happy.

Reply:
Do you mean place these outside of the component or rely on Svelte's own component typing to manage prop types?

Reply:
No sorry it wasn't clear what line i referred to. `defaultChecked` is what i refer to.

Reply:
Ah, my bad. I'll remove that too.